apps/projects/contracts/Projects.sol

```diff
@@ -25,7 +25,7 @@ interface Bounties {
     ) external; //{}
 
     /**
-     * @notice `_sender` will update their fulfillment for `_bountyId` on Standard Bounties with the following info: `_data`
+     * @notice `_sender` updated their fulfillment for issue #`_bountyId` with the following info: `_data`
```

Comment:
Should this be past-tense? It's _about to happen_ at the time the user gets this message, right?

Reply:
Yeah, these are actually user facing and wont be forwarded.

Maybe it should just be
> Update fulfillment for issue #`_bountyId` with the following info: `_data`